hack/make-rules/test.sh

```diff
@@ -69,6 +69,10 @@ KUBE_COVER_REPORT_DIR="${KUBE_COVER_REPORT_DIR:-}"
 # coverage mode.
 KUBE_COVERPROCS=${KUBE_COVERPROCS:-4}
 KUBE_RACE=${KUBE_RACE:-}   # use KUBE_RACE="-race" to enable race testing
+# The idiomatic way to disable test caching explicitly is to use -count=1
+# Run each test and benchmark n times
+# xref: https://golang.org/pkg/cmd/go/internal/test/
+KUBE_TESTCOUNT=${KUBE_TESTCOUNT:-10}
```

Comment:
Um. I feel like this should default to 1. Optimize for local contributor speed of dev. A periodic stress job can non-default this to 10

Reply:
This should default to unset. We can't afford to run this 10 times. Setting it even to `1` busts the cache / requires tests to run always (at which point we might as well just use `-race` instead).

Reply:
(also +1 to running this with a > 1 count in a periodic job, this has been on my backburner)